core/src/main/java/org/apache/spark/util/collection/TimSort.java

```diff
@@ -788,7 +788,7 @@ private void mergeLo(int base1, int len1, int base2, int len2) {
           if (--len1 == 1)
             break outer;
           minGallop--;
-        } while (count1 >= MIN_GALLOP | count2 >= MIN_GALLOP);
+        } while (count1 >= MIN_GALLOP || count2 >= MIN_GALLOP);
```

Comment:
Thank you. Actually, this was a copy of Android TimSort.java. I'm just curious if Android change like your proposal recently?
- https://android.googlesource.com/platform/libcore/+/android-6.0.1_r77/luni/src/main/java/java/util/TimSort.java#749

Reply:
Is the same in master: https://android.googlesource.com/platform/libcore/+/refs/heads/master/ojluni/src/main/java/java/util/TimSort.java#769